shell/platform/tizen/BUILD.gn

```diff
@@ -152,12 +150,14 @@ template("embedder") {
       "capi-ui-efl-util",
       "dlog",
       "ecore",
+      "ecore_evas",
       "ecore_imf",
       "ecore_imf_evas",
       "ecore_input",
       "efl-extension",
       "eina",
       "elementary",
+      "evas",
       "feedback",
       "tbm",
       "tdm-client",
```

Comment:
`tdm-client`, `EGL`, and `GLESv2` might be moved to `libs` at line 198.

```patch
@@ -160,10 +160,7 @@ template("embedder") {
       "evas",
       "feedback",
       "tbm",
-      "tdm-client",
       "wayland-client",
-      "EGL",
-      "GLESv2",
     ]

     if (target_name == "flutter_tizen_common") {
@@ -197,7 +194,10 @@ template("embedder") {

       libs += [
         "ecore_wl2",
+        "tdm-client",
         "tizen-extension-client",
+        "EGL",
+        "GLESv2",
       ]
     }
```

Reply:
If you're okay, can you please merge this patch and make this fix in the future?

Reply:
Okay. Could you make the change as early as possible?

Reply:
Sorry. I will back to the office next week. i will do it then